File: 2752-sum-multiples/sum-multiples.dart
class Solution {
  int sumOfMultiples(int n) {
    int res = 0;
    for(;n>0;n--){
      if(n%3==0) res += n;
      else if (n%5==0) res += n;
      else if (n%7==0) res += n;
    }
    return res;
  }
}

File: 2752-sum-multiples/sum-multiples.rs
impl Solution {
    pub fn sum_of_multiples(n: i32) -> i32 {
        let mut res: i32 = 0;
        for i in 0..=n {
            if i % 3 == 0 || i % 5 == 0 || i % 7 == 0 {
                res+=i;
            }
        }
        res
    }
}